src/Scripts/Character_Scripts/Character.gd

```diff
@@ -114,8 +114,14 @@ func ui_accept_pressed():
   if(interact_areas.size() != 0 and interact_areas.back() != ""):
 	  DialogueEngine._beginTransmit(interact_areas.back())
 	
-func change_sequenced_follow_formation(formation: String):
+func change_follow(formation: String):
 	self.party_data["sequence_formation"] = formation
+	
+func change_speed(speed_string: String):
+	var default_speed: float = 60
```

Comment:
Should make default speed a field rather than a local var so it shows up at the top. Then make var speed = default speed so it's only defined that one place.